### PR TITLE
Fix quickstart workflow not to run by matrix

### PR DESCRIPTION
**Pull request by Warashi**
**What this PR does / why we need it**:

Move steps, which creates PR to update quickstarts, as another job.
Without this PR, the step runs multiple times using matrix runs.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

- **How are users affected by this change**:
- **Is this breaking change**:
- **How to migrate (if breaking change)**:

### CODE REVIEW

**Review by t-kikuc — DISMISSED** [review]

File: .github/workflows/publish_image_chart.yaml [hunk 1]
```diff
@@ -147,14 +147,31 @@ jobs:
           labels: helmRepo=pipecd
           data: ${{ env.PIPECD_VERSION }}
 
+  update-quickstart:
```
Comment (khanhtc1202) [inline .github/workflows/publish_image_chart.yaml:150]:
```suggestion
  release-quickstart-manifests:
```
How about rename like this?

Comment (Warashi) [reply .github/workflows/publish_image_chart.yaml:150]: I think its better. Thank you.

**Review by khanhtc1202 — APPROVED** [review]
Thanks 👍

**Review by t-kikuc — APPROVED** [review]
better name

**Pull request merged**